projects/1.20/assets/powah-rearchitected/powah/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+    "wiki.powah.ender_gate": "末影道门",
+    "wiki.powah.ender_gate_0": "在相邻方块与末影网络之间转移能量。",
+    "wiki.powah.ender_gate_1": "与末影单元不同，你不能通过末影道门升级各个频道。",
+    "wiki.powah.energizing": "充能",
+    "wiki.powah.energizing_0": "充能台可被用于为物体充能。其周围%s格内必须有至少1个充能棒。充能速度取决于充能棒的数量及其等级。",
+    "wiki.powah.energizing_1": "充能台本身不需要能量供应，但充能棒必须位于线缆或FE存储方块上方以工作。",
+    "wiki.powah.energizing_2": "被放置时充能台和充能棒会自动互相连接，你也可以通过手持设置为链接模式的扳手右击线缆来改变其转移模式。",
+    "wiki.powah.energy_blocks": "功能性方块",
+    "wiki.powah.energy_cable": "能量线缆",
+    "wiki.powah.energy_cable_0": "线缆可被用于转移能量。",
+    "wiki.powah.energy_cable_1": "你可以通过手持设置为配置模式的扳手右击线缆来改变其转移模式。",
+    "wiki.powah.energy_cell": "能量单元",
+    "wiki.powah.energy_cell_0": "能量单元可被用于存储FE能量。",
+    "wiki.powah.energy_cell_1": "同样可以在末影单元中通过按下Shift单击被加入至末影频道，其中所含能量也会被保留。",
+    "wiki.powah.energy_discharger": "能量提取器",
+    "wiki.powah.energy_discharger_0": "能量提取器可被用于提取已含能量物品中的能量并将之存储于自身的缓存中。可以用线缆将能量提取器中的能量提取以再次利用这些能量。",
+    "wiki.powah.energy_hopper": "能量漏斗",
+    "wiki.powah.energy_hopper_0": "能量漏斗可被用于充能包含可交互物品栏且不含FE能量的相邻方块中的可充能物品。",
+    "wiki.powah.furnator": "熔炉发电机",
+    "wiki.powah.furnator_0": "熔炉发电机通过燃烧木头、煤炭等燃料产出FE能量。",
+    "wiki.powah.furnator_1": "更高等级拥有更快的输出速度，同时每刻产出的FE能量也更多，产出能量相同时消耗燃料也更快。",
+    "wiki.powah.generators": "能量产出",
+    "wiki.powah.items": "物品",
+    "wiki.powah.lens_of_ender": "末影透镜",
+    "wiki.powah.lens_of_ender_0": "右击装至太阳能板上时可以使之无视上方方块产能。",
+    "wiki.powah.magmator": "岩浆发电机",
+    "wiki.powah.magmator_0": "岩浆发电机通过熔岩等高温流体产出FE能量。",
+    "wiki.powah.magmator_1": "更高等级拥有更快的输出速度，同时每刻产出的FE能量也更多，产出能量相同时消耗燃料也更快。",
+    "wiki.powah.materials": "材料",
+    "wiki.powah.player_aerial_pearl": "玩家天线珍珠",
+    "wiki.powah.player_aerial_pearl_0": "用于合成玩家供电仪。",
+    "wiki.powah.player_aerial_pearl_1": "对僵尸或尸壳使用天线珍珠以获得。",
+    "wiki.powah.player_transmitter": "玩家供电仪",
+    "wiki.powah.player_transmitter_0": "玩家供电仪可被用于充能链接至的玩家物品栏中的物品。装有绑定卡时可为在当前维度的玩家充能，而装有绑定卡（跨维度）时可为在任意维度的玩家充能。",
+    "wiki.powah.reactor": "反应堆",
+    "wiki.powah.reactor_0": "反应堆是一种以晶质铀为燃料的多方块FE产能器。",
+    "wiki.powah.reactor_1": "要想建造一个反应堆，你需要将36个反应堆方块放入快捷栏中，并准备一块3*4*3格大小的可建造区域。在该区域的中心放置一个反应堆方块后，它会自动完成建造。",
+    "wiki.powah.reactor_2": "反应堆会在产能时升温，此时它会更快地消耗燃料，每刻产出的FE也会减少，所以你需要水作为流体冷却剂为之降温，同时你也可以使用冰、干冰等固体冷却剂进行额外的冷却；固体冷却剂生效需要流体冷却剂作为前置条件。",
+    "wiki.powah.reactor_3": "保证反应堆的缓存充满燃料以使其更好地产出能量。",
+    "wiki.powah.reactor_4": "包括木头、煤炭在内的含碳材料可以提升燃料效益，同时使反应堆升温180 C。",
+    "wiki.powah.reactor_5": "红石可以提升能量产出与燃料消耗速度，同时使反应堆升温120 C。",
+    "wiki.powah.reactor_6": "自动模式下，反应堆会在能量缓存满的时候停止并在能量少于70%时启动。",
+    "wiki.powah.solar_panel": "太阳能板",
+    "wiki.powah.solar_panel_0": "太阳能板在阳光存在时会产出FE能量。更高等级每刻产出的FE能量更多。任何位于太阳能板上方阻挡其阳光的方块都会使其停止产出。",
+    "wiki.powah.storage_transfer": "能量存储/转移",
+    "wiki.powah.thermo_generator": "热力发电机",
+    "wiki.powah.thermo_generator_0": "热力发电机在熔岩等高温流体或方块上方时会产出FE能量。需要消耗水作为流体冷却剂。",
+    "wiki.powah.uraninite": "晶质铀矿石",
+    "wiki.powah.uraninite_0": "晶质铀矿石是一种稀有的矿石。其贫瘠变种生成于Y=64以下，普通变种生成于Y=20以下，致密变种生成于Y=0以下，矿团大小为1-5。",
```

| **审阅级别** |
|:--------------:|
|⚪ **轻微** / 不阻止合并但需三思|

原文：
- **Uraninite Ore is an ore rarely found underground at levels below 64 for poor, below 20 for the normal, and below 0 for dense, and is found in 1 - 5 block deposits.**
```suggestion
    "wiki.powah.uraninite_0": "晶质铀矿石是一种稀有的矿石。其贫瘠变种生成于Y=64以下，普通变种生成于Y=20以下，致密变种生成于Y=0以下，矿团大小为1-5个方块。",
```